modules/gapi/src/backends/streaming/gstreamingbackend.cpp

```diff
@@ -408,6 +434,12 @@ void GOCVUV::Actor::extractRMat(const cv::MediaFrame& frame, cv::RMat& rmat)
             });
             break;
         }
+        case cv::MediaFormat::GRAY:
+        {
```

Comment:
warning should be here about not consistent formats...

Reply:
What for? There is no any costly conversions here.